File: 1003.py


def fib(n):
	global zc, oc
	if n == 0:
		zc += 1 
		return 0
	elif n == 1:
		oc += 1
		return 1
	else:
		return fib(n-1)+fib(n-2)

l = int(input())
for i in range(0, l):
	zc = oc = 0
	n = int(input())
	fib(n)
	print(zc, oc)

File: 1003.cpp
#include <iostream>

using namespace std; 

int zc;
int oc; 

int fib(int n) {
	if(n == 0) {
		zc += 1;
		return 0;
	} else if(n == 1) {
		oc += 1;
		return 1;
	}
	return (fib(n-1) + fib(n-2));
}

int main() {
	zc = oc = 0;
	int n, m;
	cin >> m;

	for(int i=0; i<m; i++) {
		cin >> n;	
		fib(n);
		cout << zc << " " << oc << endl;
		zc = oc = 0; 
	}

	return 0;
}